Lib/concurrent/futures/process.py

```diff
@@ -72,15 +73,39 @@
 # workers to exit when their work queues are empty and then waits until the
 # threads/processes finish.
 
-_threads_queues = weakref.WeakKeyDictionary()
+_threads_sentinels = weakref.WeakKeyDictionary()
 _global_shutdown = False
 
+
+# This constant controls the maximal wakeup. If a job is submitted to the
+# Executor, it might take up to _POLL_TIMEOUT for the executor to notice and
+# start launching the job. This _POLL_TIMEOUT is to be cumulated with the
+# communication overhead.
+_POLL_TIMEOUT = .001
```

Comment:
I don't understand this comment. This `_POLL_TIMEOUT` only impacts  the `queue_management_thread` right? This thread is only involved in handling the results and terminating the workers, not in launching jobs.